.openpublishing.redirection.json

```diff
@@ -25,6 +25,10 @@
             "redirect_url": "/dotnet/core/deploying/index"
         },
         {
+            "source_path": "docs/core/docker/visual-studio-tools-for-docker.md",
+            "redirect_url": "/aspnet/core/publishing/visual-studio-tools-for-docker"
```

Comment:
do you wanna move the comments with the article or make them disappear?

Reply:
If possible, I'd like to move the comments. Please advise on how this is done.

Reply:
Sure! Then add the redirect_document_id like some other entries have.

Reply:
This is done. Thanks for the help!